src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +40,90 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ??? // here is my greatest solution
+def not(b: Boolean): Boolean =
+  if b then false
+  else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+def and(left: Boolean, right: Boolean): Boolean =
```

Comment:
`and` should be eager and should work in the same way as `&&` and `right` should not be evaluated if `left` is false. This `and` doesn't work in case of `and(false, loop)`. Please review the first lecture on Scala.

Do the same improvements for `or`.